website/docusaurus.config.js

```diff
@@ -111,10 +111,27 @@ var siteSettings = {
       },
       items: [
         {
-          to: "/docs/introduction",
           label: "Docs",
           position: "left",
-          activeBaseRegex: "docs/(?!(dbt-cloud))",
+          items: [
+            {
+              label: "Product Docs",
+              to: "/docs/introduction",
+              activeBaseRegex: "docs/(?!(dbt-cloud))",
```

Comment:
@john-rock do you happen to know what this `activebaseRegex` was doing?

Reply:
Had to look that up too. It looks like it [adds the active link styling](https://docusaurus.io/docs/api/themes/configuration#navbar-link) to the nav item. In this case it would apply the active link styles for any path that starts with `docs` but is not directly followed by `dbt-cloud`. 

But it looks like we have a redirect setup on `docs/dbt-cloud` paths to point to `docs/get-started-dbt` so I don't think it's doing anything 🤔 .

Reply:
Gotcha, the docs repo is full of old ancient artifacts 😅